etc/ci/check_dynamic_symbols.py

```diff
@@ -0,0 +1,26 @@
+#/usr/bin/env python
+
+import sys, re, subprocess
+from sets import Set
```

Comment:
Sets are built-in since Python 2.6 and you don't need to import them. See
https://docs.python.org/2/library/stdtypes.html#set and https://docs.python.org/3/library/stdtypes.html#set.